device/globals/global_vm.gd

```diff
@@ -28,6 +28,9 @@ var cam_target = null
 var cam_speed = 0
 var camera
 var camera_limits
+var clamp_data = []  # [was the camera clamped, camera position]
+var camera_clamped  # clamp_data[0]
+var pos  # clamp_data[1] and other target camera positions
```

Comment:
It looks like these are only used in `update_camera()`. Should be local.

Reply:
`pos` is used elsewhere, but my actual reason for defining these up here is that I don't know how Godot does variables and how expensive they are. Defining these variables on every update cycle felt like extra overhead.

Does this make any sense? Do you still want me to move them? Because in general I just want this in ;)

Reply:
I think we should prioritize readability over performance until performance becomes a problem, so  I prefer locally scoped variables.

Reply:
Fair enough, I'll remove the comment from `pos` and move the other two down. Expect a force-push hopefully soon.